tests/smokeTest.js

```diff
@@ -0,0 +1,29 @@
+// smokeTest.js
+describe('Auction app smoke test', function() {
```

Comment:
The name of the test inside the describe() should be the same as the one in the TC document.